recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+  # https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg12897.html
+  export LIBS="-lnetcdf -lhdf5 -lhdf5_hl $LIBS"
+else
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export CPPFLAGS="-I$PREFIX/include $CPPFLAGS"
+
+./configure --prefix=$PREFIX \
+            --enable-shared \
+            --enable-static \
+            --enable-extra-example-tests \
+            --disable-doxygen \
+            --enable-netcdf-4 \
+            --enable-ncgen4 \
+            --enable-dap \
+            --enable-hdf4
+
+make
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+eval ${LIBRARY_SEARCH_VAR}=$PREFIX/lib make check || cat ncdump/test-suite.log
```

Comment:
Also, I made a mistake here. Please update this to `|| { cat ncdump/test-suite.log; exit 1; }`.